lib/config/defaults.js

```diff
@@ -537,6 +537,7 @@ const applyOutputDefaults = (
 	D(output, "webassemblyModuleFilename", "[hash].module.wasm");
 	D(output, "publicPath", "");
 	D(output, "compareBeforeEmit", true);
+	D(output, "charset", true);
```

Comment:
I think we can set `false` for webpack@5, but let's @sokra answer

Reply:
I would leave it `true` by default. This has the highest compatibility. It might be deprecated in modern browsers, but this doesn't mean it's unneeded for non-modern browser.